spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/config/ConfigServerProperties.java

```diff
@@ -72,6 +73,11 @@
 	 */
 	private Encrypt encrypt = new Encrypt();
 
+	/**
+	 * Flag to indicate whether labeled directories should be ignored while identifying locations.
+	 */
+	private boolean ignoreLabeledDirectories = false;
```

Comment:
Remove this from `ConfigServerProperties`. `NativeEnvironmentRepository` is a `@ConfigurationProperties`.